data/examples/shaders/filter/rounded-rect.effect

```diff
@@ -0,0 +1,85 @@
+#define IS_FILTER
```

Comment:
License and Copyright missing.

Reply:
What License? Yours or mine? I don't really mind, so please feel free to add any license you want.

Reply:
It needs your license for this file, or rather the original authors license as this is derivative work. In some areas of the world, a lack of a license header doesn't automatically put it under the same license as the rest of the code, and so far I have not yet adopted a CLA/CAA.